.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
Change 3.13 to 3.14.

https://github.com/john-bampton/john-bampton.github.io/blob/main/.github/workflows/update-faces.yml#L23C1-L24C1